File: test/fixtures/output/php/http1/cookies.php
<?php

$request = new HttpRequest();
$request->setUrl('https://httpbin.org/cookies');
$request->setMethod(HTTP_METH_GET);

$request->setCookies([
  'bar' => 'baz',
  'foo' => 'bar'
]);

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}

File: src/targets/java/nethttp/fixtures/cookies.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://httpbin.org/cookies"))
    .header("cookie", "foo=bar; bar=baz")
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());